k8s/mariadb-galera/Makefile

```diff
@@ -1,157 +1,50 @@
 include ../crd.Makefile
 include ../gcloud.Makefile
 include ../var.Makefile
+include ../images.Makefile
 
 APP_ID ?= mariadb-galera
 VERIFY_WAIT_TIMEOUT = 1800
 
 # The digest should be updated automatically, or tag <major>.<minor>.<path/build>
 # should be available on GCP Marketplace.
 # Use `.build/mariadb-galera/VERSION` target to fill the RELEASE variable.
-MARIADB_TAG ?= sha256:a732af2cfccacefbfdc563a46366546af559fa0c81b33c08cc08ccb95fbce3de
+
+TRACK ?= 10.3
 METRICS_EXPORTER_TAG ?= v0.5.1
 PEER_FINDER_TAG ?= latest
 
-BUILD_ID := $(shell date --utc +%Y%m%d-%H%M%S)
-TRACK ?= 10.3
-RELEASE ?= 10.3.17-$(BUILD_ID)
+# List of images used in application
+TARGET_IMAGES := mysqld-exporter prometheus-to-sd peer-finder
 
-$(info ---- TRACK = $(TRACK))
-$(info ---- RELEASE = $(RELEASE))
+# Image names should correspond with TARGET_IMAGES list
+image-$(APP_ID) ?= $(call get_sha256,$(SOURCE_REGISTRY)/mariadb10:$(TRACK))
```

Comment:
I have a question: I cannot see that `SOURCE_REGISTRY` var is defined above. Does it work and how?

Reply:
It works, because images are defined as callbacks to get_sha256 function, so it is actually executed later, when c2d_deployer file is already included and rule is actually called.

But I agree that it is not the best in terms of readability, so I will move `SOURCE_REGISTRY` definition to this file.